### ISSUE TITLE
Improve import times by making imgui_fig optional (slow matplotlib import)

### ISSUE THREAD

**Issue by Tremeschin** [pthom/imgui_bundle#280]
Hi, I used [tuna](https://github.com/nschloe/tuna) to check import times with the command below and got the following result on latest pypi (1.5.2), cp312:

`PYTHONPROFILEIMPORTTIME=1 python -c "import imgui_bundle" 2> /tmp/file && tuna /tmp/file`

![image](https://github.com/user-attachments/assets/30fb7d9d-b155-4d9e-8857-ab57923a62f7)

By commenting out the following on imgui_bundle init file, the import times are much more bearable:

```python
# if matplotlib is not installed, we can't import imgui_fig
try:
    from imgui_bundle import imgui_fig as imgui_fig
except ImportError:
    pass
```

![image](https://github.com/user-attachments/assets/879f465a-1443-4136-baeb-b3831e314231)

It's honestly fair tanking numpy, pydantic, glfw import times and probably some bootstrapping of imgui in your cython code. These 283ms of difference in import times are _very_ noticeable when running CLI applications that happens to just import imgui but not use it immediately. Furthermore, showing matplotlib figures on the windows seems like a niche use case that one would explicitly and optionally import `imgui_fig` part of the code for 🙂 

I can hack myself out of it without changing your init by pretending I don't have matplotlib as I'm not using it:

```python
# Same avg 155 ms to run the imports
import sys

sys.modules["matplotlib"] = None

import imgui_bundle
```

Could this be improved on the main package? I didn't quite follow where it's located in the source tree, or if it was already solved and it's just pending a new build for a release<sup>With also cp313 😅?</sup>

Thanks!

**Comment by pthom** [comment]
Hi @Tremeschin 

This is fixed in e66da2bb

A new  release will come soon (with python 3.13  of course!)

**Comment by Tremeschin** [comment]
Awesome @pthom! the choice and implementation for lazy loading looks adequate on the commit; I didn't look enough for that file and thought it was imported within cython, could have made a better original comment knowing it now 😄

Thanks for the package again, it works great, any further improvements or issues I'll be sure to contribute!
